### PR TITLE
logger: add hack to ensure log lines aren't empty in Kibana

### PR DESCRIPTION
**Pull request by bmwill**
Closes: #6200

### CODE REVIEW

File: common/logger/src/libra_logger.rs [hunk 1]
```diff
@@ -245,20 +245,26 @@ impl LoggerService {
...
-            }
-
             if let Some(printer) = &self.printer {
                 let s = format(&entry).expect("Unable to format");
                 printer.write(s)
             }
+
+            if let Some(writer) = &mut writer {
+                Self::write_to_logstash(writer, entry);
+            }
```
Comment (davidiw) [inline common/logger/src/libra_logger.rs:255]: interesting, I hadn't realized that these were two different things... should we actually have a Vec of printers?

**Review by davidiw — APPROVED** [review]

**Comment by davidiw** [comment]
Thanks man!

File: common/logger/src/libra_logger.rs (hunk 1 above)
Comment (bmwill) [reply common/logger/src/libra_logger.rs:255]: Yeah we send things out to stderr as well as to a TCP endpoint.

**Pull request closed without merging**